CHANGELOG.md

```diff
@@ -434,9 +465,6 @@ aws lambda invoke --function-name $PREFIX-data-migration1 \
 
 ### Changed
 
-- **CUMULUS-NONE**
```

Comment:
This was removed as a dupe in the main branch as part of the 10.1.1 release updates.